resources/js/src/app/helper/featureDetect.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Synchronous function to detect if WebP images are supported.
+ *
+ * @param browser object delivered by detect-browser
+ * @returns {boolean}
+ */
+export function detectWebP(browser)
```

Comment:
Woher kommt diese Logik? Gibts dazu eine Quelle? Google selbst stellt eine andere Methode zur Feature-Erkennung zur Verfügung: https://developers.google.com/speed/webp/faq#how_can_i_detect_browser_support_for_webp

Reply:
https://stackoverflow.com/questions/5573096/detecting-webp-support
Ich habe den Code aus der ersten Antwort adaptiert. Ich habe mich bewusst gegen Googles Lösung entschieden, da deren Lösung asynchron ist und potentiell einen Network Request beinhaltet. 
Die Lösung über Canvas fand ich deutlich eleganter, und sie ist synchron. 
Ich habe sie in Chrome, Firefox und Safari getestet.

Reply:
Einen Network-Request sehe ich da nicht: Die Testbilder werden alle Inline bereitgestellt. Modernizr setzt ebenfalls auf eine asynchrone Lösung. Ich würde mich hier eher auf Google und Modernizr als auf einzelne Antworten bei stackoverflow verlassen...

https://github.com/Modernizr/Modernizr/blob/400db4043c22af98d46e1d2b9cbc5cb062791192/feature-detects/img/webp.js

Reply:
Done.